src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -4391,6 +4391,13 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
                 tmp = ASR::make_EnumValue_t(al, x.base.base.loc, enum_member_var,
                         enum_t, enum_member_variable->m_type,
                         ASRUtils::expr_value(enum_member_variable->m_symbolic_value));
+            } else if (ASR::is_a<ASR::Module_t>(*t)) {
+                ASR::Module_t *m = ASR::down_cast<ASR::Module_t>(t);
+                ASR::symbol_t *sym = import_from_module(al, m, current_scope, value,
+                                    x.m_attr, x.m_attr, x.base.base.loc);
+                LFORTRAN_ASSERT(ASR::is_a<ASR::ExternalSymbol_t>(*sym));
+                current_scope->add_symbol(x.m_attr, sym);
```

Comment:
What if its already imported and added? We will add it to the current scope even though its already there?

Reply:
I think that here we should change the name of the symbol too. If we are already using some variable with the same name then it might be a conflict, for example:
```py
import math

def f():
    pi: f64 = 4.5030
    print(math.pi, math.e)

f()
```
This gives an error:
```
semantic error: pi already defined
```

To tackle this, we need to change the symbol name to `module_name@attr_name`.

> We will add it to the current scope even though its already there?

`import_from_module` ensures that the symbol is not present in the `current_scope`.

Reply:
> To tackle this, we need to change the symbol name to `module_name@attr_name`.

Correct, name mangling is needed here I think. @certik Thoughts?

> `import_from_module` ensures that the symbol is not present in the `current_scope`.

Please test this. Try using `math.pi` twice.

Reply:
Done!